### PR TITLE
Houdini/update docs to 0.2.12

### PR DESCRIPTION
**Pull request by MustafaJafar**
# Changelog description
Update Houdini Docs to be the same as current Houdini addon version `0.2.12`

### CODE REVIEW

**Comment by MustafaJafar** [comment]
This PR update Houdini docs to align with `0.2.12` although the branch name points to `0.2.11`.
I did that because it's much easier to do so instead of making another PR that requires this PR.

File: website/docs/addon_houdini_artist.md [hunk 1]
```diff
@@ -376,6 +379,42 @@ When the render is finished, all the caches would be located in your publish fol
...
 ![Houdini Farm Per Task](assets/houdini/artist/houdini_farm_cache_loader.png)
 
+### Publishing render to Deadline
+
+Rendering on the farm is supported for:
+- **Karma ROP**
+- **Mantra ROP**
+- **Redshift ROP**
+- **Arnold ROP**
+- **VRay ROP**
```
Comment (moonyuet) [inline website/docs/addon_houdini_artist.md:389]:
```suggestion
The following renderers are supported for both local and farm renderings:
- **Karma**
- **Mantra**
- **Redshift **
- **Arnold**
- **VRay**
```

Comment (MustafaJafar) [reply website/docs/addon_houdini_artist.md:389]: Nice Idea. I think after applying your suggestion, we won't be able to merge this PR until the local render support PR merged. https://github.com/ynput/ayon-core/pull/328

Comment (MustafaJafar) [reply website/docs/addon_houdini_artist.md:389]: Let's mark it as resolved temporarily. 
We can make another PR to update Houdini docs when new features are added.

**Review by moonyuet — APPROVED** [review]

**Pull request merged**